micrometer/src/main/java/feign/micrometer/MicrometerCapability.java

```diff
@@ -38,6 +43,17 @@ public MicrometerCapability(MeterRegistry meterRegistry) {
     this.meterRegistry = meterRegistry;
   }
 
+  public MicrometerCapability(MeterRegistry meterRegistry, List<Tag> tagsList) {
+    meterRegistry.config().commonTags(tagsList);
+    this.meterRegistry = meterRegistry;
+  }
+
+  public MicrometerCapability(MeterRegistry meterRegistry, Map<String, String> tagsMap) {
+    List<Tag> tagsList = mapTags(tagsMap);
+    meterRegistry.config().commonTags(tagsList);
```

Comment:
Do you think this would be testable under our current tests?

Reply:
@BartekBanachowicz what about this?

Reply:
For now, the answer from my side is no. Not because it isn't doable, but because I don't know how to achieve it yet. I'm still digging through existing test classes